### ISSUE TITLE
Setting {breaks: false} does nothing

### ISSUE THREAD

**Issue by Venryx** [HHogg/remarkable-react#7]
Title says it all. 

Using the code:
```
var md = new Remarkable({breaks: false});
md.renderer = new RemarkableReactRenderer({breaks: false});
```

The resulting html still has `<br/>` inserted for any "\n" in the `source`.

**Comment by Swivelgames** [comment]
@Venryx Just after perusing the codebase, as a temporary solution until @HHogg takes a look at this, you might try overwriting `softbreak` token with a `null` component?

```javascript
md.renderer = new RemarkableReactRenderer({
    components: {
        softbreak: () => null
    }
});
```

**Comment by HHogg** [comment]
Thanks for reporting this @Venryx. I'm struggling to replicate this to find where the issue is, please could you post some example input markdown?

**Comment by HHogg** [comment]
Scratch that last comment, I see the issue now.

**Comment by HHogg** [comment]
Also thanks @Swivelgames for taking a look at this 😺 I've sent up #8. @Venryx please could you confirm this is the expected behaviour.

**Comment by Venryx** [comment]
Yep, that's exactly as wanted. : )   (and how other Markdown systems render it)

Thanks for such a quick fix!

**Comment by HHogg** [comment]
Awesome thanks for checking and no problem... a patched release is now available under [v0.2.1](https://github.com/HHogg/remarkable-react/releases/tag/v0.2.1)
